sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_utils.py

```diff
@@ -69,7 +69,9 @@ def _get_azure_sdk_target_source(attributes: Attributes) -> Optional[str]:
     if attributes:
         # New semconv attributes: https://github.com/Azure/azure-sdk-for-python/pull/29203
         # TODO: Keep track of when azure-sdk supports stable semconv for these fields
-        peer_address = attributes.get("net.peer.name") or attributes.get("peer.address")
+        peer_address = (
```

Comment:
Thanks for this change. Could you remove the # TODO in the line above this, seeing as we are supporting stable semconv now?